### PR TITLE
Add intro text

### PR DESCRIPTION
**Pull request by as-dr**
Added some intro text, modeled off what we have for the Grants page. Hoping @amy-jung and @Davidutro can help finesse this.

image

### CODE REVIEW

File: src/pages/meetups.js [hunk 1]
```diff
@@ -4,65 +4,98 @@ import styled from "styled-components"
...
+      {/* Primary Stats Section */}
+      <IntroHeader>
+        We've brought together{" "}
+        <IntroHeaderHighlight>
+          {meetupsData.totalNumberReportedAttendees}
+        </IntroHeaderHighlight>{" "}
+        people since February 2018.
+      </IntroHeader>
+      <IntroContent>
+        MakerDAO’s Core Community Development Initiative supports physical
```
Comment (amy-jung) [inline src/pages/meetups.js:33]:
```suggestion
        MakerDAO’s Core Community Development Initiative supports virtual and physical
```

**Review by MaximumCrash — APPROVED** [review]
This looks good, need to commit Amy's requested changes.

**Comment by as-dr** [comment]
On it

**Comment by as-dr** [comment]
@MaximumCrash Good to go.

**Pull request merged**